pkg/crc/machine/bundle/metadata.go

```diff
@@ -182,6 +182,8 @@ func (bundle *CrcBundleInfo) GetBundleType() crcPreset.Preset {
 		return crcPreset.OpenShift
 	case "podman", "podman_custom":
 		return crcPreset.Podman
+	case "microshift", "microshift_custom":
+		return crcPreset.Microshift
```

Comment:
Might be nicer as `type_str := strings.TrimSuffix(bundle.Type, "_custom")` and then the switch/case becomes
```
switch type_str {
  case "snc":
  case "podman:
 ...
 ```